addons/mrp/migrations/13.0.2.0/post-migration.py

```diff
@@ -14,12 +15,14 @@ def fill_bom_product_template_attribute_value(env):
         (mrp_bom_line_id, product_template_attribute_value_id)
     SELECT mbl_pav_rel.mrp_bom_line_id, ptav.id
     FROM mrp_bom_line_product_attribute_value_rel mbl_pav_rel
+    JOIN product_attribute_value pav ON pav.id = mbl_pav_rel.product_attribute_value_id
     JOIN mrp_bom_line mbl ON mbl.id = mbl_pav_rel.mrp_bom_line_id
     JOIN mrp_bom mb ON mbl.bom_id = mb.id
     JOIN product_template pt ON mb.product_tmpl_id = pt.id

```

Comment:
useless join we need only `id` which we already have in `mb.product_tmpl_id`

Reply:
that's https://github.com/OCA/OpenUpgrade/pull/2957 😆